config.js

```diff
@@ -0,0 +1,6 @@
+var config = {};
+
+config.GCMessage = 'ABCDGC';
+config.MCMessage = 'ABCDMC';
+
+module.exports = config;
```

Comment:
Think about use Buffers in order to be able to play with encodings, bytearrays, hex...
https://nodejs.org/docs/latest-v4.x/api/buffer.html

(casi seguro que no hace falta... y son un buen lío, pero para los parseos y para montar mensajes intuyo que van a venir bien... asi que... pégate un poco)